src/types.ts

```diff
@@ -4324,6 +4324,10 @@ export interface StayComp extends Comp {
 	 * If the obj should not be destroyed on scene switch.
 	 */
 	stay: boolean,
+	/**
+	 * Array of scenes that the obj will stay on.
+	 */
+	scenes: string[],
```

Comment:
Can you rename this to something like `scenesToStay`? The property will be assigned to the game object directly, `obj.scenes` is a bit too ambiguous on what it does

Reply:
Fair point, will do that real quick

Reply:
Thank you!